src/util/address.rs

```diff
@@ -129,6 +129,14 @@ impl Shl<usize> for Address {
     }
 }
 
+/// Default constructor
+impl Default for Address {
```

Comment:
The same issue here as the default zero block for `Block`. I suggest we remove them.

Reply:
removed